docs/docs-source/docs/modules/get-started/pages/setup-example-project-configure-build.adoc

```diff
@@ -56,11 +56,11 @@ include::{cloudflow-examples-version}@docsnippets:ROOT:example$sensor-data-scala
 +
 The script is a standard Scala sbt build file--with the addition of the Cloudflow plugin for Akka Streams, the Cloudflow application plugin, and also the Scalafmt plugin that we suggest to keep the style consistent in the application (optional).
 +
-. Create the file `project/cloudflow-plugins.sbt` and add the Cloudflow plugin dependency in it:
+. Create the file `project/cloudflow-plugins.sbt` add logic to get the correct version, and add the Cloudflow plugin dependency:
```

Comment:
Rather specify the exact version instead of using the `lastestVersion` trick used to easily work with examples for build.

Reply:
Could just inline this piece in antora and use a version variable to template it?

Reply:
@RayRoestenburg, I'm not sure how to do this, there is a {cloudflow-version} variable in antora.yml, but the value is set to "dev". I think the build script might change this, but I can't see whether it would end up being the latest version or not? And, should it be latest, or the latest stable version?

Reply:
Not sure if we still have a variable for now, you can hardcode it for now as well to 2.0.23 and we sort it out later.

Reply:
Sorry it is `{cloudflow-version}`

Reply:
(It is an asciidoc attribute)

Reply:
I changed it to the variable and committed, but when I build locally, the variable resolves to `dev`, as you can see below. Can I assume that for production, the build substitutes the correct value?

![image](https://user-images.githubusercontent.com/22889339/107071791-e2141400-67aa-11eb-9cca-dadc509031af.png)

Reply:
correct, place the attribute in quotes and keep just the last line (e.g. `addSbtPlugin...`) thanks @rstento !

Reply:
Done, will do the same for the commercial doc.